modules/installation-and-upgrade/pages/container-deployment/suma/server-deployment-vmdk-suma.adoc

```diff
@@ -0,0 +1,320 @@
+[[install-vm-vmware]]
+= Deploy {productname} {productnumber} Server as a Virtual Machine - VMware
+ifeval::[{uyuni-content} == true]
+:noindex:
+endif::[]
+
+This chapter provides the required Virtual Machine settings for deployment of {productname} {productnumber} as an Image.
+VMware will be used as a sandbox for this installation.
+
+
+// FIXME: make this section a snippet or move it to a general intro for both,
+//        KVM and VMware
+== Available Images
+
+[IMPORTANT]
+====
+The preferred method for deploying {productname} {productnumber} Server is to use one of the following available images.
+All tools are included in these images greatly simplifying deployment.
+====
+
+Images for {productname} {productnumber} are available at link:https://www.suse.com/download/suse-manager/[{productname} {productnumber} VM images].
+
+[NOTE]
+====
+For more information on preparing raw images see:
+
+* link:https://documentation.suse.com/en-us/sle-micro/5.5/single-html/SLE-Micro-deployment/#sec-raw-preparation[]
+* link:https://documentation.suse.com/en-us/sle-micro/5.5/single-html/SLE-Micro-deployment/#cha-images-procedure[]
+
+For additional information on the self install images see:
+
+* link:https://documentation.suse.com/en-us/sle-micro/5.5/single-html/SLE-Micro-deployment/#cha-selfinstal-procedure[]
+====
+
+.Available Server Images
+[cols="3, 3", options="header"]
+|===
+| Architecture | Image Format
+
+| aarch64| qcow2, vmdk
+
+| x86_64  | qcow2, vmdk, raw, Self Installer
+
+| ppc64le | raw, Self Installer
+
+| s390x * | qcow2, raw
+|===
+
+__* Two storage options are available for s390x: CDL DASD and FBA.__
+
+
+
+// FIXME: we probably do not need this section
+
+[[sumavm.ignition]]
+== {productname} basic configuration using [systemitem]``Ignition``
+
+[IMPORTANT]
+====
+{productname} image does not set up [systemitem]``root`` or any other user account.
+User or [systemitem]``root`` authentications need to be set up during first boot.
+This can be done using [systemitem]``Ignition`` or [systemitem]``Cloud-Init`` methods.
+====
```

Comment:
5.0 and now also 4.3 has firstboot setup enabled.

Similarly like in KVM scenario, when ignition/combustion is not provided it will ask for password, etc.

Can we move this whole ignition/combustion/cloud-init to a separate page we can reference from other deployments?

Reply:
Ok, but for the moment, I'll remove ignition/combustion here, because it is optional if I understand it correctly.  If needed, we could add hints about such an advanced configuration option later.